sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -39,6 +39,24 @@ namespace Azure { namespace Core {
    */
   class Context {
   public:
+    /**
+     * @brief A context key.
+     */
+    struct Key final
+    {
+      Key const* m_uniqueAddress;
```

Comment:
Question: If we just remove this member, will everything still work? e.g. if we define two instances of empty struct, is it guaranteed that they will have different addresses?

Reply:
Further question: if we replace this member with `bool never_used = false`. Is it guaranteed that compiler won't optimize this member out and two instances will always have different addresses?

Reply:
If we create as global static instance, addresses should be different - it is exported as two different symbols.